libs/blocks/card/card.js

```diff
@@ -115,9 +117,11 @@ const init = (el) => {
   section.classList.add('milo-card-section');
   const row = el.querySelector(':scope > div');
   const picture = el.querySelector('picture');
-  const links = el.querySelectorAll('a');
   const styles = Array.from(el.classList);
   const cardType = getCardType(styles);
+  const merch = el.classList.contains('merch') && cardType === HALF;
```

Comment:
Nit: Use `styles` rather than calling `el.classList` a second time
```suggestion
  const merch = styles.includes('merch') && cardType === HALF;
```